core/src/main/java/org/opensearch/sql/expression/NamedExpression.java

```diff
@@ -60,6 +60,6 @@ public <T, C> T accept(ExpressionNodeVisitor<T, C> visitor, C context) {
 
   @Override
   public String toString() {
-    return getNameOrAlias();
+    return Strings.isNullOrEmpty(alias) ? name : name + " AS " + alias;
```

Comment:
Looks like you should add a new method `getNameWithAlias()` in this class similar to `getNameOrAlias()`.
And I have some concerns when to use `getNameOrAlias()` and when to use `getNameWithAlias()`?
Could you deep dive in it?

Reply:
`getNameOrAlias` are used to get the field name to use in other operator. While in its `toString`, I want to add more info than the field name.

I think we don't add such a new function as  it will only be used in this `toString` method,  we should add it unless it can be reused in other places.

Reply:
> we should add it unless it can be reused in other places as well.

I doubt are there any other places to replace to `getNameWithAlias` either as `toString` does. Can you help to confirm them?

Reply:
Having scanned over all the usage of `getNameOrAlias`, I don't see any other places need to be replaced. They are all related to execution.

Reply:
Thanks @qianheng-aws .  @dai-chen Can u help to double confirm it since u are the author of this class?

Reply:
Sorry I can't recall the specific details. I think it should be safe to change the `toString` method only.